src/Makefile

```diff
@@ -18,8 +18,11 @@ endif
 
 ## FILE-SPECIFIC MODIFICATIONS
 
+# Recipe doesn't work, using explicit rule instead
+# frontend.out: state.h mode_id.h cursor.o fe_modes.o
+frontend.out: frontend.c frontend.h state.h mode_id.h cursor.o fe_modes.o
```

Comment:
This works if the header files are removed, and `frontend.c` is included by default.
```suggestion
frontend.out: cursor.o fe_modes.o
```